### PR TITLE
fix get attachment lost nil key

### PR DESCRIPTION
**Pull request by champly**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes get attachment lost nil key, if the value is empty, get attachment will lost it

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: request.go [hunk 1]
```diff
@@ -23,9 +23,7 @@ import (
 	"strconv"
 	"strings"
 	"time"
-)
```
Comment (AlexStocks) [inline request.go:26]: pls do not delete it.

File: request.go [hunk 2]
```diff
@@ -23,9 +23,7 @@ import (
...
 
-import (
```
Comment (AlexStocks) [inline request.go:28]: pls do not delete it.

File: request.go (hunk 1 above)
Comment (AlexStocks) [reply request.go:26]: pls add it manually by yourself.

Comment (champly) [reply request.go]: OK，i manually import it

**Comment by AlexStocks** [comment]
@champly pls add an unit test for this pr.

**Comment by champly** [comment]
> @champly pls add an unit test for this pr.

already add unit test.  PTAL

File: request_test.go [hunk 3]
```diff
@@ -80,3 +81,37 @@ func TestDescRegex(t *testing.T) {
 	assert.Equal(t, "[Ljava/lang/String;", results[0])
 	assert.Equal(t, "[I", results[1])
 }
+
+func TestToMapStringString(t *testing.T) {
```
Comment (AlexStocks) [inline request_test.go:85]: maybe u can create an issue for this pr, #1113 for example. And then, u can rename this test func name to TestIssue1113.

**Comment by champly** [comment]
#192

**Pull request merged**